src/main/java/club/issizler/okyanus/api/perms/PermissionService.java

```diff
@@ -38,4 +38,22 @@
      */
     void removePermission(Permissible permissible, String perm);
 
+    /**
+     * Sets whether or not the permissible is an operator,
+     * or otherwise holds all permissions.
+     *
+     * @param op {@code true} to set to an operator,
+     * {@code false} to set to a non-operator.
+     */
+    void setOp(Permissible permissible, boolean op);
```

Comment:
I am not entirely sure if we should merge OP-ness with permissions, as the OP permission is sort of "internal" to the game. and permission services would need to access MC internals to get if the player is actually OP or not.